frontend/base/src/test/java/com/virtuslab/gitmachete/resourcebundles/junit/GitMacheteBundlePropertiesTestSuite.java

```diff
@@ -28,4 +29,44 @@ public void htmlProperties_should_have_correct_syntax() throws IOException {
           assertEquals("HTML property key should have a .HTML suffix", ".HTML", key.substring(key.length() - 5));
         });
   }
+
+  @Test
+  public void properties_should_have_no_double_quote_wrapping() throws IOException {
+
+    Properties property = new Properties();
```

Comment:
```suggestion
    Properties properties = new Properties();
```
here and in the next test